### ISSUE TITLE
jquery 11 breaks some Grav core

### ISSUE THREAD

**Issue by charlesknightsbridge** [mikahanninen/grav-plugin-facebook#22]
This broke contenttabs, which requires jquery ui-tab; not sure why, but core Grav installs jquery 1.2, so perhaps some conflict there.

**Comment by mikahanninen** [comment]
Which version of Grav do you have ? I will check this later today - I have some sports for few hours first.

**Comment by charlesknightsbridge** [comment]
I was using the most recent production release of 1.1.2. The problem persisted when I upgraded to 1.2
I have attached two screen grabs showing the loaded scripts on my test page with and without Facebook plugin activated.

With Facebook plugin:
<img width="291" alt="withfacebookplugin" src="https://cloud.githubusercontent.com/assets/855040/24579589/61f30134-16c6-11e7-8326-37a7aa32debc.png">

Without Facebook plugin:
<img width="273" alt="withoutfacebookplugin" src="https://cloud.githubusercontent.com/assets/855040/24579590/620030fc-16c6-11e7-8fa5-20d85214965f.png">

Here is the javascript error I get with the plugin installed:

TypeError: jQuery('#g-contenttabs-contenttabs-9329').tabs is not a function. (In 'jQuery('#g-contenttabs-contenttabs-9329').tabs', 'jQuery('#g-contenttabs-contenttabs-9329').tabs' is undefined)
